client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx

```diff
@@ -264,31 +241,19 @@ const ClinicalDetails: React.FC<Props> = ({ id }: Props): JSX.Element => {
-                                                onChange={(event, selectedStreet) => props.onChange(selectedStreet ? selectedStreet.id : '')}
-                                                onInputChange={(event, selectedStreetName) => {
-                                                    if (event?.type !== 'blur') {
-                                                        setIsolationStreetName(selectedStreetName);
-                                                        if (selectedStreetName === '') {
-                                                            props.onChange('');
-                                                            methods.setValue(`${ClinicalDetailsFields.ISOLATION_ADDRESS}.${ClinicalDetailsFields.ISOLATION_STREET}`, '');
-                                                        }
-                                                    }
+                                                options={Array.from(streetsInCity, ([id, value]) => ({ id, value }))}
+                                                getOptionLabel={(option) => {
+                                                    if (option) {
+                                                        if (option?.value) return option.value?.displayName
+                                                        else return '';
+                                                    } else return option
                                                 }}
+                                                value={props.value && {id: props.value as string, value: streetsInCity.get(props.value) as Street}}
+                                                onChange={(event, selectedStreet) => props.onChange(selectedStreet ? selectedStreet.id : '')}
                                                 renderInput={(params) =>
                                                     <TextField
                                                         test-id='currentQuarantineStreet'
-                                                        error={methods.errors[ClinicalDetailsFields.ISOLATION_ADDRESS] && methods.errors[ClinicalDetailsFields.ISOLATION_ADDRESS][ClinicalDetailsFields.ISOLATION_STREET] ? true : false}
-                                                        label={
-                                                            methods.errors[ClinicalDetailsFields.ISOLATION_ADDRESS] &&
-                                                            methods.errors[ClinicalDetailsFields.ISOLATION_ADDRESS][ClinicalDetailsFields.ISOLATION_STREET] ?
-                                                            methods.errors[ClinicalDetailsFields.ISOLATION_ADDRESS][ClinicalDetailsFields.ISOLATION_STREET].message
-                                                                :
-                                                                'רחוב'
-                                                        }
+                                                        label='רחוב'
```

Comment:
why removing the conditions in case of an error

Reply:
street is not required